### PR TITLE
Add get_jdbc_df function

### PR DESCRIPTION
**Pull request by ianlee0609**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-this is commtent exercise
\ No newline at end of file
+this is commtent exercise
```
Comment (scu19930609) [inline README.md]: delete this line

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
...
+Add get_date_str function
```
Comment (scu19930609) [inline README.md]: more detail

**Review by scu19930609 — COMMENTED** [review]
please fixed

**Comment by ianlee0609** [comment]
please chek it out

**Pull request merged**